orca-bakery/src/test/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTaskSpec.groovy

```diff
@@ -118,7 +118,7 @@ class CreateBakeTaskSpec extends Specification {
 
   @Shared
   def buildInfo = new JenkinsBuildInfo(
-    "name", 0, "http://jenkins", "SUCCESS",
+    "name", null, "http://jenkins", "SUCCESS",
```

Comment:
Wouldn't `"0"` be a more straightforward conversion?  Though I'm all for testing with null too.

Reply:
i could refactor to treat `"0"` as null/missing?

Reply:
I think it's fine to use null to indicate missing...but then [this comment](https://github.com/spinnaker/orca/pull/4316#discussion_r1005038701) says -1 used to mean missing....but from [here](https://github.com/spinnaker/orca/pull/4316/files#r1020612013) and specifically the `buildInfo.getNumber() > 0` part...it's not only -1, is everything <= 0 that's considered missing...so this is fine.  Thanks for reading :)